wallet/src/de/schildbach/wallet/ui/dashpay/DashPayViewModel.kt

```diff
@@ -29,13 +29,10 @@ class DashPayViewModel(application: Application) : AndroidViewModel(application)
 
     private val platformRepo = PlatformRepo(application as WalletApplication)
 
-
     private val usernameLiveData = MutableLiveData<String>()
-    private val platformStatusLiveData = MutableLiveData<Boolean>()
 
     // Job instance (https://stackoverflow.com/questions/57723714/how-to-cancel-a-running-livedata-coroutine-block/57726583#57726583)
     private var getUsernameJob = Job()
-    private var isPlatformAvailableJob = Job()
```

Comment:
This field and all related code is only needed if you want to be able to reload the value of related `LiveData` multiple times and each time cancel previous request is it wasn't yet finished. I'll try to explain it in more details on Slack.